### ISSUE TITLE
After reboot, Wyze sensors unavailable

### ISSUE THREAD

**Issue by fitzgeraldbrady** [kevinvincent/ha-wyzesense#131]
Hi, everything's been working fine until a HA reboot today, now all Wyze sensors show as unavailable, and I get a new error in my logs at startup:

Hardware: Intel NUC
OS: Ubuntu Server 20.04
Docker: Yes

Home Assistant Core 0.107.7 (auto-updates off)
Wyze Sense Component Installed: 0.0.7 (This is the version I started with)

configuration.yaml entry:

```
binary_sensor:
  - platform: wyzesense
    device: auto
```

Logs:
2020-05-02 12:58:40 WARNING (MainThread) [homeassistant.loader] You are using a custom integration for hacs which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant.

2020-05-02 12:58:42 WARNING (MainThread) [homeassistant.loader] You are using a custom integration for wyzesense which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant.

(This is the new error)
2020-05-02 12:58:43 ERROR (MainThread) [homeassistant.components.binary_sensor] Error while setting up wyzesense platform for binary_sensor
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/entity_platform.py", line 179, in _async_setup_platform
    await asyncio.wait_for(asyncio.shield(task), SLOW_SETUP_MAX_WAIT)
  File "/usr/local/lib/python3.7/asyncio/tasks.py", line 442, in wait_for
    return fut.result()
  File "/usr/local/lib/python3.7/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/config/custom_components/wyzesense/binary_sensor.py", line 74, in setup_platform
    config[CONF_DEVICE] = findDongle()
  File "/config/custom_components/wyzesense/binary_sensor.py", line 65, in findDongle
    df = subprocess.check_output(["ls", "-la", "/sys/class/hidraw"]).decode('utf-8').lower()
  File "/usr/local/lib/python3.7/subprocess.py", line 411, in check_output
    **kwargs).stdout
  File "/usr/local/lib/python3.7/subprocess.py", line 512, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['ls', '-la', '/sys/class/hidraw']' returned non-zero exit status 1.

**Comment by fitzgeraldbrady** [comment]
Update:

I noticed that the bridge had a solid yellow light, I believe that is is normally blue, right?

Anyways, after several reboots it just started working again... now bridge light is blue and contact sensors can be seen.

Weird!

**Comment by gterpstra75** [comment]
> I noticed that the bridge had a solid yellow light, I believe that is is normally blue, right?

That is correct.    I have similar conditions that you have described in the past months a few times.  If I lose the blue light and it doesn't connect on the reboot, I usually unplug and replug the wyzesense hub and then completely reboot ubuntu/docker.   That has usually always resolved the issue.   I actually haven't had this situation in over a month now.    I don't know if it because I have updated the firmware on the wyzesense hub or something else.

**Comment by oriolism** [comment]
I had this issue too, the problem was that my NUC doen't detect hidraw device. The fix, connect to bridge to usb cable extension or a usb hub.

**Comment by jjhampton** [comment]
I had the same issue. I'm running Home Assistant on a Ubuntu VM in VirtualBox, and the host machine rebooted. After Home Assistant started back up, blue light was ON on the Wyze Bridge, but nothing from the sensors registered in Home Assistant.

Unplugged bridge, and plugged it back in. It started out with a yellow light. 
Rebooted the VM completely and restarted Home Assistant. 
Bridge light turned blue.
Wyze sensors now appear to be working and triggering state updates in Home Asssistant.

**Comment by dj69** [comment]
FWIW the creator of the HACS integration spoke to this issue in one of the readme files.  The sensors need to have their state "tweaked" or "toggled".  What does this mean?  In HA, under Developer Tools and then States, look for the wyzesense states and set them to *anything vs the "undetected" state.   I set mine to "detected" but the real states are either "on" or "off". I then disconnected the wyzesense bridge (mine is directly connected to my HA raspberry pi via USB), reconnected it (yellow light) and then restarted HA.  Then my wyzesense sensors all said "off" rather than my previous state or "undetected".  I walked through my house and everything was detected.  All back to normal.
